lib/internal/bootstrap/node.js

```diff
@@ -297,6 +290,41 @@ function startup() {
     }
   });
 
+  // Set up coverage exit hooks.
+  let originalReallyExit = process.reallyExit;
+  // Core coverage generation using nyc instrumented lib/ files:
+  // see `make coverage-build`.
+  // This does not affect user land.
+  if (global.__coverage__) {
```

Comment:
similarly, we should soon be in a position to drop this code. CC: @mhdawson

Reply:
Until that actually happens, I guess a TODO would be enough?